projects/plugins/boost/app/assets/src/js/react-components/sections/header/index.tsx

```diff
@@ -0,0 +1,55 @@
+import classNames from 'classnames';
+import React from 'react';
+import { navigate } from '../../../utils/navigate';
+import BackButton from '../../components/back-button';
+import ChevronRight from '../../svg/chevron-right';
+import Logo from '../../svg/logo';
+import styles from './styles.module.scss';
+
+interface HeaderProps {
```

Comment:
I'm still fuzzy on the details, but I feel we should be consistent when declaring types. In the back-button, we use `type`, and here `interface`.

Reply:
Updated https://github.com/Automattic/jetpack/pull/33987/commits/efb87f2e847d447cc16c1abb63f6100426b42c8c